src/documents/jqUnit.md

```diff
@@ -0,0 +1,305 @@
+---
+title: jqUnit
+layout: default
+category: Infusion
+---
+
+Bundled with infusion is ***jqUnit***, a library modelled on the [xUnit](https://en.wikipedia.org/wiki/XUnit) API style which wraps base functionality provided by the
+popular [QUnit](https://qunitjs.com/). All of the QUnit base functionality remains available in a jqUnit fixture at the `QUnit` namespace, and you should consult the QUnit documentation
+for details of packaging markup-based fixtures and its own API semantic. However, jqUnit is a complete wrapper and it is possible and recommended to write complete
+test suites without reference to QUnit. jqUnit is based on the 1.x API of QUnit, and not the (currently unreleased) 2.x version of QUnit which is API incompatible with QUnit 1.x.
+
+As well as use in the browser, there is also a (currently unstable) node.js module, [node-jqunit](https://github.com/fluid-project/node-jqUnit) which allows the use of the same
+testing API for writing node.js tests.
+
+A more advanced piece of infrastructure is the [IoC Testing Framework](#IoCTestingFramework.md) which is useful for writing asynchronous test fixtures targetting Infusion's IoC
+[component trees](UnderstandingInfusionComponent.md). This is not a replacement for jqUnit but a library layered on top of it which is good for certain specialised purposes. If you are
+writing plain unit tests, as well as integration tests which don't have a highly asynchronous, conversational style, you should continue to write jqUnit fixtures. If you find you are
+writing large-scale integration or acceptance tests against significantly-sized parts of an application, that require sequences of asynchronous conversation, for example simulating user
+GUI interaction or HTTP requests, you should use the IoC Testing Framework.
+
+# jqUnit API
+
+## Organising and controlling fixtures
+
+### jqUnit.module(name, hooks)
+
+Starts a group of related tests which will display with the module's name as a prefix. Direct passthrough for [QUnit.module](http://api.qunitjs.com/QUnit.module/)
+
+### jqUnit.test(name, testFunc)
+
+Registers (queues) a synchronous test fixture by providing a callback which will run it. Direct passthrough for [QUnit.test](http://api.qunitjs.com/QUnit.test/). This is equivalent
+to a call to `jqUnit.asyncTest`, where the fixture ends with `jqUnit.start`.
+
+### jqUnit.asyncTest(name, testFunc)
+
+Registers (queues) a synchronous test fixture by providing a callback which will run it. Direct passthrough for [QUnit.asyncTest](http://api.qunitjs.com/QUnit.asyncTest/).
+
+### jqUnit.start()
+
+Restarts QUnit's progress through its fixtures, which were previously suspended by a call to `jqUnit.stop()`. The idiom is that whilst the system is suspended by `stop`, the
```

Comment:
Would you have any objection to replacing all the "whilst"s with "while"? Not critical, but my brain snags on them...